rust_snuba/src/arroyo_utils.rs

```diff
@@ -0,0 +1,17 @@
+use rust_arroyo::processing::strategies::InvalidMessage;
+/// Some helper functions that work around Arroyo's ergonomics, and should eventually make it into
```

Comment:
@untitaker curious why this logic was added/ported to Snuba and maybe not Arroyo?

Reply:
@ayirr7 it's a bunch of things that should eventually be moved to arroyo, but might require further thought. I don't think this function that I put here is generic enough to be put into arroyo as-is, but something *like this* should exist in arroyo.

In this case I'd like arroyo to have some way to generate InvalidMesage from a Message more easily, but I can't come up with a good design right now. I think this function is not one.